doc/man/cockpit.conf.xml

```diff
@@ -74,6 +74,42 @@ Origins = https://somedomain1.com https://somedomain2.com:9090
     </variablelist>
   </refsect1>
 
+  <refsect1 id="oauth">
+	  <title>oauth</title>
```

Comment:
We are not case insensitive in cockpit_conf. We are case insensitive in our authentication mechanisms in that we normalize they type to lower case before doing anything with it.

So outcome, make cockpit_conf be case insensitive and make everything here camelcase?